crypto/provider.c

```diff
@@ -13,10 +13,16 @@
 #include <openssl/core_names.h>
 #include "internal/provider.h"
 
-OSSL_PROVIDER *OSSL_PROVIDER_load(OPENSSL_CTX *libctx, const char *name)
+OSSL_PROVIDER *OSSL_PROVIDER_load(OPENSSL_CTX *libctx, const char *name,
+                                  unsigned int flags)
 {
     OSSL_PROVIDER *prov = NULL;
 
+    /* An attempt to load a provider disables auto-loading of defaults */
+    if (!(flags & OSSL_PROVIDER_LOAD_FLAG_RETAIN_FALLBACK)
+            && !ossl_provider_disable_fallback_loading(libctx))
```

Comment:
Is indentation on this line correct?

Reply:
It is, it's got the optional extra indent after the if.  However, a few lines below the other form is used.  I've made them consistent.